config.json

```diff
@@ -2070,8 +2070,8 @@
       {
         "groupId": "com.google.android.material",
         "artifactId": "material",
-        "version": "1.10.0",
-        "nugetVersion": "1.10.0.2",
+        "version": "1.11.0",
```

Comment:
here you also need to downgrade material

Reply:
> here you also need to downgrade material

Thanks. I know. I have slightly bigger problems with `dotnet` crashing. See error below.